CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.4.0 (2020-04-08)
+
+- Updated the autocomplete and lookup components to implement the affix and overlay services. [#74](https://github.com/blackbaud/skyux-lookup/pull/74)
```

Comment:
Does the lookup component actually get these services? I see them imported into the autocomplete module, but I don't see a corresponding import for lookup.